src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -693,6 +693,18 @@ components:
           type: integer
           format: int64
 
+    CursorLimitMixin:
+      type: object
+      properties:
+        cursor:
+          description: This cursor allows forward pagination, by providing the cursor from the previous request.
+          type: string
+          nullable: true
+        limit_per_page:
```

Comment:
should we specify upper limit in specs maybe and default value if not provided?

Reply:
hmm you're right, we return meaningful error response, so should be ok. Is that documented anywhere or returned from some api "configuration alike" endpoint? or api user has to check it manually what is the limit?

Reply:
What the limit is isn't exposed anywhere and that is a OK thing I believe. It is optionally up to the NG owner to document that in some handcrafted docs (think: user guide)